tests/network/service_mesh/conftest.py

```diff
@@ -441,3 +441,8 @@ def vmi_http_server(vm_fedora_with_service_mesh_annotation):
         vm=vm_fedora_with_service_mesh_annotation,
         commands=[f'while true ; do  echo -e "HTTP/1.1 200 OK\n\n $(date)" | nc -l -p {SERVICE_MESH_PORT}  ; done &'],
     )
+
+
+@pytest.fixture(scope="module")
+def label_istio_injection_namespace(namespace, admin_client):
```

Comment:
Maybe in the spirit of what @sbahar619 suggested, you can use this one to wrap the namespace one.
I.e. rename this to `istio_namespace`, add the label like you do here and return the namespace back.

Reply:
Sounds like an overkill to me.
Going this way will require changing all the namespace references in this module from `namespace` to `istio_namespace`, which contradicts the behavior in all the other tests in this repo.
Can you please explain the rational for doing so?

Reply:
OK, I see the conceptual point you are raising. I'll change it.
BTW - `namespace` fixture may not be used or referenced in the test module, but it is in the package's conftest. Nevertheless I'll apply this change.